LiteCore/Query/QueryParser.cc

```diff
@@ -598,8 +598,30 @@ namespace litecore {
         for (auto &ftsTable : _indexJoinTables) {
             auto &table = ftsTable.first;
             auto &alias = ftsTable.second;
+            auto idxAt = table.find(KeyStore::kIndexSeparator);
```

Comment:
Below this area could stand some comments about the logic

Reply:
Agreed and will do.